docs/src/main/asciidoc/scheduler.adoc

```diff
@@ -109,16 +109,18 @@ public class CounterBean {
 (Quarkus is using 10 worker threads for the scheduler). If it is not available the method invocation should be re-scheduled by default i.e
 it should be invoked as soon as possible. The invocation of the scheduled method does not depend on the status or result of the previous invocation.
 4. The code is pretty straightforward. Every 10 seconds, the counter is incremented.
-5. Define a job with a cron-like expression. The syntax is currently based on http://www.quartz-scheduler.org/documentation/quartz-2.3.0/tutorials/crontrigger.html[Quartz Cron Trigger]. The annotated method is executed at 10:15am every day.
+5. Define a job with a cron-like expression. The annotated method is executed at 10:15am every day.
 6. Define a job with a cron-like expression `cron.expr` which is configurable in `application.properties`.
 
 == Updating the application configuration file
 
 Edit the `application.properties` file and add the `cron.expr` configuration:
 [source,shell]
 ----
+quarkus.scheduler.cron-type=quartz <1>
```

Comment:
Hm, `quartz` is the default value. So I wonder if we should comment out this line and add one more line above with _"Define the cron syntax to use..."_?

Reply:
You are suggesting that we remove this config line?

Reply:
Not removing, but something like this instead:
```properties
# By default, the syntax used for cron expressions is based on Quartz
# You can change the syntax using the following property:
# quarkus.scheduler.cron-type=unix

cron.expr=*/5 * * * * ?
```

WDYT?